jujucrashdump/crashdump.py

```diff
@@ -207,7 +207,11 @@ def get_all(self):
         juju_status = yaml.load(open('juju_status.yaml', 'r'),
                                 Loader=yaml.FullLoader)
         for machine, machine_data in juju_status['machines'].items():
-            machines[machine] = machine_data['ip-addresses']
+            try:
+                machines[machine] = machine_data['ip-addresses']
+            except KeyError:
+                # A machine in allocating may not have an IP yet.
+                pass
```

Comment:
I think it would be safer to just continue here, rather than pass, so we don't try to process containers on this machine either.

Reply:
Made this change.